### ISSUE TITLE
Incompatible with minecells

### ISSUE THREAD

**Issue by hideki2k02** [qouteall/ImmersivePortalsMod#1312]
### According to your testing, does the issue occur with ONLY Immersive Portals mod and Fabric API?

No

### Describe the issue

Hello, i've used the mod beforehand with AOF 1.3.0 and it worked fine (I've followed the guide, gotten the right Sodium Version downgraded Iris to 0.4 and used IF 2.2.5), but after updating to 1.5.1 the mod suddenly no longers works.

I've double checked, even tried copying the jar from the other instance, this crash only happens if Immersive Portals is enabled for some reason, but i've not been able to figure out why. I've also tried running IF without Iris and it also crashes, maybe its a mod incompatibility?

LWJGL Version: 3.2.2
Fabric Version: 0.14.18
Minecraft Version: 1.19.2

Immersive Portals Version: Tried both 2.2.5 and 2.3.1
Iris Version: 1.4.0

https://mclo.gs/0H3ztbP
https://pastebin.com/K9VS6sVJ

[latest.log](https://github.com/iPortalTeam/ImmersivePortalsMod/files/11117563/latest.log)

**Comment by hideki2k02** [comment]
My bad, i've forgot to include the crash log

[crash-2023-03-30_23.17.12-client.txt](https://github.com/iPortalTeam/ImmersivePortalsMod/files/11117589/crash-2023-03-30_23.17.12-client.txt)

**Comment by qouteall** [comment]
Seem to be a mixin conflict with xlpackets

**Comment by hideki2k02** [comment]
I've tried playing with only these mods and things work fine, so its probably not it...?
- Fabric 0.14.18
- Fabric API 0.76.0
- Immersive Portals 2.3.1
- Iris 1.4
- Sodium 0.44+build.18
- XL Packets 1.19.2-4

Also i've managed to play with the latest Immersive Portals (seems to work if you're using Iris 1.4)
[latest.log](https://github.com/iPortalTeam/ImmersivePortalsMod/files/11123185/latest.log)

**Comment by qouteall** [comment]
Oh it seem to be related to minecells
```
[23:17:12] [Netty Local Client IO #0/WARN]: @ModifyConstant conflict. Skipping imm_ptl.mixins.json:common.networking.MixinClientboundCustomPayloadPacket from mod imm_ptl_core->@ModifyConstant::modifySizeLimitWhenReadingPacket(I)I with priority 1000, already redirected by minecells.mixins.json:network.CustomPayloadS2CPacketMixin from mod minecells->@ModifyConstant::minecells$init2ModifyMaxSize(I)I with priority 2275
```

**Comment by qouteall** [comment]
mixin conflict: https://github.com/mim1q/MineCells/blob/6f995b86e00bb1c8bbea86ca91e5cd7e54fce44c/src/main/java/com/github/mim1q/minecells/mixin/network/CustomPayloadS2CPacketMixin.java#L24

**Comment by hideki2k02** [comment]
It does seem so, AOF 6 (1.3.0) was using Mine Cells 1.3.3, however the problem seems to be version 1.3.5

![image](https://user-images.githubusercontent.com/5614680/229264423-de221f94-250e-479e-ae8c-f7c21c39c0b6.png)

I'm not entirely sure if that was the problem, i'm not sure what a mixin is, But it was fixed after version 1.4.0. I'm not sure if a "workaround" is required then haha
![image](https://user-images.githubusercontent.com/5614680/229264383-c149bf4c-45b0-4431-856e-5986e4cdff65.png)

**Comment by qouteall** [comment]
So updating minecells can fix it. Issue closed.
